homework-submission/week1-react/src/todo.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import changeDateFormat from './change-format-date.js';
+
+class Todo extends React.Component {
+
+    render () {
+        const {id, text, date, done, changeDone, index} = this.props;
+
+        return(
+                <li id={'todo'+id}>
```

Comment:
No use for the id in your `Css`, so no need for it here 👌

Reply:
you are right, it is done :)